lib/srv/db/server.go

```diff
@@ -253,6 +264,7 @@ func New(ctx context.Context, config Config) (*Server, error) {
 		dynamicLabels:    make(map[string]*labels.Dynamic),
 		heartbeats:       make(map[string]*srv.Heartbeat),
 		proxiedDatabases: config.Databases.ToMap(),
+		limiter:          config.Limiter,
```

Comment:
Same comment as in app server - I'm not sure having a separate field is needed, just use limiter from config.

Reply:
Fair enough. Removed